EcommerceStore.API/Controllers/AuthenticationController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using EcommerceStore.API.Authentication;
+using EcommerceStore.Application.Exceptions;
+using EcommerceStore.Application.Interfaces;
+using EcommerceStore.Application.Models;
+using EcommerceStore.Application.Models.ViewModels;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using System.Threading.Tasks;
+
+namespace EcommerceStore.API.Controllers
+{
+    [Route("api/account")]
+    [ApiController]
+    public class AuthenticationController : ControllerBase
```

Comment:
naming inconsistency, why is the route named "account" and controller has different name?